lite/Changelog

```diff
@@ -1,6 +1,38 @@
 Stable versions
 ---------------
 
+4.5.0 ():
+	- make xmp_set_position() consistently clear pattern break/jump vars
+	- fix S3M ADPCM4 samples
+	- xmp_get_format_list() now returns const char* const*, not char**
+	  (no ABI change)
+	- xmp_test_module, xmp_load_module, xmp_set_instrument_path and
+	  xmp_smix_load_sample() now accept const char* path parameters
+	  (no ABI change)
+	- explicitly document that callers of xmp_load_module_from_file()
+	  are responsible for closing their own file.
+	- remove nonportable use of fdopen in xmp_load_module_from_file()
+	- fix a seek issue with xmp_load_module_from_memory
+	- fix memory-io functions' error handling
+	- fix lite build mod loader symbols
+	- fix IT message length
+	- fix strictness of MOD pattern data tester
+	- fix loading of XMLiTE XM modules
+	- handle XM 16-bit samples with odd in-file data
+	- fix smix sample allocation
+	- force reset of buffer state on player start
+	- fix double free in case of ADPCM sample load error
```

Comment:
See the above comment re: ADPCM loading being disabled for libxmp-lite.

Reply:
Updated the lite-side of the patch now.